frontend/assets/javascripts/src/modules/identityPopup.js

```diff
@@ -53,9 +54,11 @@ define([
         var windowLocation = window.location;
         var currentUrl = windowLocation.pathname + windowLocation.search;
 
-        IDENTITY_MENU_CTA_ELEM.setAttribute('href',
-            populateReturnUrl(IDENTITY_MENU_CTA_ELEM.getAttribute('href'), currentUrl)
-        );
+        if(IDENTITY_MENU_CTA_URL) {
```

Comment:
Split the selector used for URL replacing from the toggle selector so I can just opt-out of replacing the url.

(As an aside if we actually passed `pageInfo` to all pages we could avoid needing to do this with JavaScript. Says the client-side developer)